### PR TITLE
Add Divide for great justice

### PR DESCRIPTION
**Pull request by yuna9**
Let's do this!

### CODE REVIEW

**Review by yhelen — CHANGES_REQUESTED** [review]
Please account for division by zero.

**Comment by elc1798** [comment]
Hi StuySplash!
I'm getting spammed by GitHub notifications. 😠

File: adder.go [hunk 1]
```diff
@@ -7,3 +7,7 @@ func Add(n1, n2 int) int {
 func Multiply(n1, n2 int) int {
 	return n1 * n2
 }
+
+func Divide(dividend, divisor int) int {
+	return dividend / divisor
```
Comment (yhelen) [inline adder.go]: Are you sure there are _zero_ problems in this code? Do we want to get to Einstein?

**Review by yhelen — APPROVED** [review]
Interesting change. :+1:

**Pull request merged**